meadServer/app.js

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
```
app.get('*', (req, res) => {
  res.sendFile(path.join(__dirname+'/client/build/index.html'));
});
```

A bit weird to always return the home page, better to redirect if you don't want to throw a 404.